### ISSUE TITLE
oc_service is unable to create with label

### ISSUE THREAD

**Issue by ewolinetz** [openshift/openshift-ansible#4086]
Using `oc_service` with a `labels` parameter is unable to correctly process the values provided

Using the following all result with "argument labels is of type <type 'list'> and we were unable to convert to dict"
```yaml
oc_service:
  ...
  labels:
  - logging-infra: 'support'
```

```yaml
  labels:
    logging-infra: 'support'
```

```yaml
  labels: [ 'logging-infra=support' ]
```

```yaml
  labels:
  - key: 'logging-infra'
    value: 'support'
```

**Comment by ewolinetz** [comment]
@kwoodson

**Comment by richm** [comment]
Have you tried
```
labels:
  logging-infra: support
  component: es
...
```
?

**Comment by ewolinetz** [comment]
should be addressed by #4091
